packages/native/test/lib/ElementAssertion.test.tsx

```diff
@@ -271,4 +271,64 @@ describe("[Unit] ElementAssertion.test.ts", () => {
       });
     });
   });
+
+  describe (".toContainElement", () => {
+    const element = render(
```

Comment:
As a good practice, elements should be rendered on each test `it(..)` function, so the cleanup function cleans what's rendered before each test, avoid flaky tests, etc.